src/mon/AuthMonitor.cc

```diff
@@ -42,34 +42,21 @@
 #define dout_prefix _prefix(_dout, mon, get_last_committed())
 using namespace TOPNSPC::common;
 
-using std::cerr;
-using std::cout;
-using std::dec;
-using std::hex;
 using std::list;
 using std::map;
 using std::make_pair;
 using std::ostream;
 using std::ostringstream;
-using std::pair;
 using std::set;
-using std::setfill;
 using std::string;
 using std::stringstream;
-using std::to_string;
-using std::vector;
-using std::unique_ptr;
 
```

Comment:
Neither of these two is being imported into current namespace (99% of times) in `src/mon`, `src/osd` and `src/mds`. Perhaps because these are not frequently used, it's easier for readers/other developers to remember them when their namespace is being explicitly mentioned. Therefore I am leaving these as it is.

Reply:
I thought the motivation was to make the code more readable and less verbose, hence I don't think that looking at whether it was ever done in other source files is an effective way to approach this task.
I never saw conflicting implementations of the classes so I don't see why one can't import them. There is one file where `using namespace std` is active and `lock_guard` is referenced unqualified: `src/client/SyntheticClient.cc`.

But I also believe that this is a minor thing so feel free to proceed as you find appropriate, my general approval is recorded.